libs/providers/flagd/README.md

```diff
@@ -42,20 +44,30 @@ Options can be defined in the constructor or as environment variables, with cons
   }))
 ```
 
-### Example Using a Unix Socket
+### Remote flag resolving using Unix Socket
 
 ```
   OpenFeature.setProvider(new FlagdProvider({
       socketPath: "/tmp/flagd.socks",
   }))
 ```
 
+### In-process resolver with gRPC sync connectivity
+
+```
+  OpenFeature.setProvider(new FlagdProvider({
+      host: 'localhost',
+      port: 8013,
```

Comment:
This may imply that the in-process provider works with flagd. Should we use the document the flagd proxy port? Perhaps we should also include a note underneither explaining where the configuration is expected to come from.

Reply:
I agree we should change the port in this example, but I think more importantly, we should put explanation and links to [this](https://flagd.dev/concepts/syncs/#grpc-sync) in the js-docs so people can be pointed in the right direction from the code.

Reply:
Good point, will update the docs and usage example